en/docs/bijira-concepts/data-planes.md

```
@@ -1,98 +1,98 @@
-
-API Platform supports flexible management models for private data planes, facilitating collaboration between WSO2 and customers across diverse scenarios.
-
-- **WSO2 Fully Managed – WSO2 Subscription:** Infrastructure and PDP are hosted under WSO2’s subscription.
-- **WSO2 Fully Managed – Customer Subscription:** Infrastructure is hosted under the customer’s subscription, but WSO2 manages the PDP.
-- **Customer Self-Managed:** WSO2 provides installation scripts and updates, while the customer maintains full control of the infrastructure and operations.
-
+# Data Planes
+
+API Platform’s architecture is built around two core components, i.e. the control plane and the data plane.
+
+### Control Plane
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)